src/views/Datasets.vue

```diff
@@ -21,51 +21,111 @@
 		<b-alert variant="danger" :show="!!error" dismissible @dismissed="error = null">{{ error }}</b-alert>
 
 		<!-- table -->
-		<b-table id="dataset-list" ref="datasetTable" :items="datasetList" :fields="fields" select-mode="single" striped hover show-empty selectable :tbody-tr-class="rowClass" filter="truthy value" :filter-function="filter" no-provider-filtering no-provider-sorting :busy.sync="isBusy" primary-key="id" :tbody-transition-props="{'name': 'datasets-flip'}">
-			<template slot="published" slot-scope="row">
-				<div class="dataset-row-publish-status">
-					<font-awesome-icon icon="circle" class="text-primary" v-if="row.item.published && !isItemPublishedAndHasUpdates(row.item)" />
-					<font-awesome-icon icon="circle" class="text-warning" v-else-if="row.item.published && isItemPublishedAndHasUpdates(row.item)" />
-					<font-awesome-icon icon="circle" class="text-success" v-else />
-				</div>
+		<b-table id="dataset-list"
+				ref="datasetTable"
+				:items="datasetList"
+				:fields="fields"
+				striped
+				:tbody-tr-class="rowClass"
+				filter="truthy value"
+				:filter-function="filter"
+				:busy.sync="isBusy"
+				primary-key="id"
+				:tbody-transition-props="{'name': 'datasets-flip'}">
```

Comment:
Wasn't this something that Jori said would cause unnecessary render?

Reply:
true that is removed in another PR